src/pybind/mgr/dashboard/frontend/src/app/shared/api/rgw-realm.service.ts

```diff
@@ -30,8 +30,14 @@ export class RgwRealmService {
     return this.http.put(`${this.url}/${realm.name}`, requestBody);
   }
 
-  list(): Observable<object> {
-    return this.http.get<object>(`${this.url}`);
+  list(replicable?: boolean): Observable<object> {
```

Comment:
We should define a model for realm (optional, you can do in another PR too)
```suggestion
  list(replicable?: boolean): Observable<Realm[]> {
```